evoting/service/service.go

```diff
@@ -111,6 +111,11 @@ func (s *Service) Open(req *evoting.Open) (*evoting.OpenReply, error) {
 		return nil, err
 	}
 
+	// sanity check - do not allow elections to be created retrospectively
+	if req.Election.End < time.Now().Unix() {
```

Comment:
We need to take care here and do an audit of the code, starting right here: if only the 'entry-point-service' checks for valid data, then we have a security problem. So all of these things should be verified in a verification function that is applied to the evoting-skipchain.

Whenever a new master- or voting-skipchain is created, the corresponding verification-functions need to be appended to the creation of the genesis-block:

https://github.com/dedis/cothority/blob/master/evoting/lib/chains.go#L12

`skipchain.VerificationStandard` should be replaced with `election.ElectionVerification` or so.